src/lib.rs

```diff
@@ -25,6 +25,11 @@
 extern crate num;
 extern crate rulinalg;
 
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate maplit;
```

Comment:
Are you still using this crate? If so, where?

Reply:
Yes, I use both these in the initialisation of the `static` `Hashmap` constants for the code look ups.